spec/controllers/users/omniauth_controller_spec.rb

```diff
@@ -10,6 +10,9 @@
           'email' => email,
           'uuid' => uuid,
         },
+        'credentials'=> {
```

Comment:
super nit, can we fix the spacing here? To add a space before the hashrocket here and on the next line? ` =>`